pkgs/development/python-modules/flask-openid/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, flask
+, python3-openid
+, python-openid
+, isPy27
+}:
+
+buildPythonPackage rec {
+  pname = "flask-openid";
+  version = "1.2.5";
+
+  src = fetchPypi {
+    pname = "Flask-OpenID";
+    inherit version;
+    sha256 = "5a8ffe1c8c0ad1cc1f5030e1223ea27f8861ee0215a2a58a528cc61379e5ccab";
+  };
+
+  propagatedBuildInputs = [
+    flask
+  ] ++ (if isPy27 then [ python-openid ] else [ python3-openid ]);
+
+  # no tests for repo...
+  doCheck = false;
```

Comment:
a lot of packages dont export tests to pypi, may want to checkout from github

Reply:
I checked on github and it seems there really are no tests: https://github.com/mitsuhiko/flask-openid/commit/83534279b9f10e615612b837738d2817b01bcf1a